### DISCUSSION TITLE
Throw Query Exception when seeding

### DISCUSSION THREAD

**Discussion by ariexx** [spatie/laravel-activitylog#1107] [Q&A]
Hi, i got problem here, when im seeding my laravel i got this error, i dont know why:

SQLSTATE[01000]: Warning: 1265 Data truncated for column 'subject_id' at row 1 (SQL: insert into `activity_log` (`log_name`, `properties`, `batch_uuid`, `ev
ent`, `subject_id`, `subject_type`, `description`, `updated_at`, `created_at`) values (default, [], ?, created, 71426fad-6a36-4ce9-80eb-079fdb321262, App\Mode
ls\User, created, 2022-11-03 16:22:45, 2022-11-03 16:22:45))

**Comment by Gummibeer** [comment]
Seems like you are using UUIDs instead of BigInts - so you should adjust the migrations to use UUIDs for the foreign keys.

**Reply by ariexx** [reply]
thank youuu mate

**Comment by anuzpandey** [comment]
How did you do it?
I have some models with `uuid` as a primary key and others are normal `int`.
So, I cannot update the migration to use `uuid` only.
